src/app/routes/utils/regex/index.test.js

```diff
@@ -443,7 +425,7 @@ describe('legacyAssetPageDataPath', () => {
   shouldNotMatchInvalidRoutes(invalidDataRoutes, legacyAssetPageDataPath);
 });
 
-describe('frontPage -> homePage migration', () => {
+describe('homepage regex on environments', () => {
```

Comment:
ukchina is included as a valid route for home page regex. We won't be merging this until the ukchina work is done.

Reply:
We've paused the merging for now so this won't be merged in this sprint.